msteams-platform/concepts/design/enable-app-customization.md

```diff
@@ -9,14 +9,17 @@ ms.topic: overview
 
 # Customize your Teams app
```

Comment:
Let's change the title to something else. 
Customize app sounds like changing functionality of the app, which is not the case.
**Feature name** = app customization. 
**High-level dev action** mentioned in this article = allow admins to do app customization by changing only metadata of the app.
**Use case** = Modify the appearance of app to suit org-specific branding.

A title should include keywords from all these 3 items.

Reply:
My Suggestion: 
1. Change metadata and customize your Teams app
2. Modify metadata and appearance of your Teams app

Reply:
Thank @v-ajiths, I am noting the suggestion in a questionnaire to PMs. I shall let a few folks vote on the titles and then choose.

Reply:
@v-ajiths , let's make the title (only for now, till we split it into different ones) reflect both the dev tasks. Let's go with `Enable app customization and block apps till admin allows` for now in the interest of time, and revisit it soon.

Reply:
Updated as suggested